examples/showcase/bachelors_degrees_by_gender.py

```diff
@@ -30,22 +30,20 @@
 fig.subplots_adjust(left=.06, right=.75, bottom=.02, top=.94)
 # Limit the range of the plot to only where the data is.
 # Avoid unnecessary whitespace.
-ax.set_xlim(1968.5, 2011.1)
+ax.set_xlim(1969.5, 2011.1)
 ax.set_ylim(-0.25, 90)
-ax.get_xaxis().get_major_formatter().set_useOffset(False)
 
 # Make sure your axis ticks are large enough to be easily read.
 # You don't want your viewers squinting to read your plot.
 plt.xticks(range(1970, 2011, 10), fontsize=14)
-plt.yticks(range(0, 91, 10), ['{0}%'.format(x)
-                              for x in range(0, 91, 10)], fontsize=14)
+plt.yticks(range(0, 91, 10), fontsize=14)
+ax.xaxis.set_major_formatter(plt.FuncFormatter('{:.0f}'.format))
+ax.yaxis.set_major_formatter(plt.FuncFormatter('{:.0f}%'.format))
```

Comment:
No, because this would multiply the entries by 100 (they are in [0, 100], not [0, 1]).